vercel.json

```diff
@@ -20,24 +14,20 @@
       "source": "/docs/settings/config/:path*",
       "destination": "/docs/configuration/radio/:path*"
     },
-    {
-      "source": "/docs/settings/moduleconfig",
-      "destination": "/docs/configuration/module"
-    },
     {
       "source": "/docs/settings/moduleconfig/",
```

Comment:
I think `/audio` isn't forwarding because you don't have `path` here. If you need subpaths here to forward you'll have to add that.

Reply:
doesn't work with trailingslash=true
https://sandbox-71lgde8mm-pdxlocations.vercel.app/docs/settings/moduleconfig/audio

works
https://meshtastic.org/docs/settings/moduleconfig/audio

Reply:
I'm not sure how this is happening because I don't see it being removed in this PR, but [L#32](https://github.com/meshtastic/meshtastic/blob/master/vercel.json#L32) of the config currently has a redirect for `moduleconfig/:path` while yours does not. I think that's why it's not working. This redirect still needs to be somewhere in your vercel.json.
```
    {
      "source": "/docs/settings/moduleconfig/:path*",
      "destination": "/docs/configuration/module/:path*"
    },
````

Reply:
I tried a bunch of variations so it may have not been changed back.  I removed a slash that I added.